### ISSUE TITLE
example in README for publishing to wiki is failing

### ISSUE THREAD

**Issue by heckj** [SwiftDocOrg/swift-doc#86]
I wanted to try out the Github action that publishing the docs in commonmark to the github wiki - I've added the job to my pipeline, and it runs "successfully", but when you look at the detail it's failed to do it's work.

The job is available at https://github.com/heckj/OpenTelemetryModels/pull/7/checks?check_run_id=598253694

with the error detail:
```
  with:
    path: Documentation
  env:
    GH_PERSONAL_ACCESS_TOKEN: ***
/usr/bin/docker run --name c1a2251720909414cf48e808fac8244e4c4_b9fe73 --label 430c1a --workdir /github/workspace --rm -e GH_PERSONAL_ACCESS_TOKEN -e INPUT_PATH -e HOME -e GITHUB_JOB -e GITHUB_REF -e GITHUB_SHA -e GITHUB_REPOSITORY -e GITHUB_REPOSITORY_OWNER -e GITHUB_RUN_ID -e GITHUB_RUN_NUMBER -e GITHUB_ACTOR -e GITHUB_WORKFLOW -e GITHUB_HEAD_REF -e GITHUB_BASE_REF -e GITHUB_EVENT_NAME -e GITHUB_WORKSPACE -e GITHUB_ACTION -e GITHUB_EVENT_PATH -e RUNNER_OS -e RUNNER_TOOL_CACHE -e RUNNER_TEMP -e RUNNER_WORKSPACE -e ACTIONS_RUNTIME_URL -e ACTIONS_RUNTIME_TOKEN -e ACTIONS_CACHE_URL -e GITHUB_ACTIONS=true -e CI=true -v "/var/run/docker.sock":"/var/run/docker.sock" -v "/home/runner/work/_temp/_github_home":"/github/home" -v "/home/runner/work/_temp/_github_workflow":"/github/workflow" -v "/home/runner/work/OpenTelemetryModels/OpenTelemetryModels":"/github/workspace" 430c1a:2251720909414cf48e808fac8244e4c4  "Documentation"
Initialized empty Git repository in /tmp/ci-XXXXkFhEhb/.git/
fatal: remote error: access denied or repository not exported: /3/nw/38/4c/f5/266259832/255487748.wiki.git
On branch master

Initial commit

nothing to commit
error: src refspec master does not match any
error: failed to push some refs to 'https://***@github.com/heckj/OpenTelemetryModels.wiki.git'
```

The wiki had *not* been previously initialized (I'd not previously created any wiki pages), so I suspect this might be a corner case in the processing of the publisher.

In either case, I suspect that since it failed to publish, the error wasn't correctly propogated to the external jobs to let it know that it didn't succeed.

<img width="965" alt="Screen Shot 2020-04-18 at 12 03 02 PM" src="https://user-images.githubusercontent.com/43388/79668853-91c0bc80-816c-11ea-87fa-444c6ed74389.png">

**Comment by heckj** [comment]
After initializing the wiki and re-running the job, it looks like it's working - but the docs aren't appearing in the wiki on github.

I also tried creating a page called Documentation in the wiki and then re-running to kit in case it required.

When I manually clone those repositories to see what's in them, I'm only seeing the raw detail that I initialized by hand into the wiki, nothing from the swift-doc output.

`git clone https://github.com/heckj/OpenTelemetryModels.wiki.git`

The most recent run that's appeared to work is online at https://github.com/heckj/OpenTelemetryModels/pull/7/checks?check_run_id=599772766

<img width="962" alt="Screen Shot 2020-04-19 at 10 24 37 AM" src="https://user-images.githubusercontent.com/43388/79694863-099fed00-8228-11ea-83eb-6fc2cb0af41e.png">

I feel like I'm missing something brutally obvious here, but I'm not cluing in to what it is.

**Comment by heckj** [comment]
it was totally my mistake - I referenced `source` rather than `sources` - which is the default. I've made a tiny PR that tweak's the README as an offering to stave this off from the next poor fool that copy/pasta's without paying attention, and included in that PR (#92) a warning emit from the generate command so that something will be visible in the Github CI Action output when that does happen.
